gateserver/main.py

```diff
@@ -0,0 +1,21 @@
+"""
+Gate server runner -- https://github.com/fmfi-svt/gate/wiki/Architecture#server
+"""
+
+from .messagehandler import MessageHandler
+import os, sys
+import socketserver as ss
```

Comment:
Vlastne ked uz sme pri tom: neviem ci je to zdokumentovane ale v socketserveri je priamo definovany ThreadingUDPServer. Netreba subclassovat.